angular.json

```diff
@@ -25,6 +24,20 @@
             "styles": [
               "./node_modules/@angular/material/prebuilt-themes/pink-bluegrey.css",
               "src/globals.scss"
+            ],
+            "vendorChunk": true,
+            "extractLicenses": false,
+            "buildOptimizer": false,
+            "sourceMap": true,
+            "optimization": false,
+            "namedChunks": true,
+            "allowedCommonJsDependencies": [
+              "@cyber4all/clark-taxonomy",
+              "aws-sdk",
+              "chart.js",
+              "md5",
+              "ng-click-outside",
+              "rxjs/internal/observable/of"
```

Comment:
What is this used for?

Reply:
We use it to catch an error on the learning objects. It's declared here because these allow require CommonJS which is no longer used. It just suppresses the error thrown

Reply:
![image](https://user-images.githubusercontent.com/55514493/150357081-f6df68c6-1f17-43ef-9a13-307fa18e6995.png)